src/main/java/seedu/address/model/AppointmentStatistics.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.model;
+
+/**
+ * Collects related statistics of appointment book.
+ */
+public class AppointmentStatistics {
+    private int doneToday;
```

Comment:
It is not a big issue but personally I feel the naming of the attributes can be clearer? Let me know what do you think. 👍